src/helpers/env.js

```diff
@@ -0,0 +1 @@
+export const isProduction = () => process.env.NODE_ENV === 'production'
```

Comment:
Not that it matters, but was just curious why you're exporting a function instead of a boolean here.

Reply:
hmm good question, don't really have a reason.
I guess it's a habit of putting everything into functions to ensure immutability, but it's obviously not needed in this case, won't matter as webpack should strip it out (I assume?)

Reply:
Don't know. I know that uglifyjs handles part of the dead code elimination, but wouldn't be surprised if it doesn't do that with functions.

Reply:
Could grep the production build for the `isProduction console.log` lines to be sure.

Reply:
just checked, it get's stipped out, it's just left in the `.js.map` file

Reply:
Ok, it doesn't, changed it now